integration/heapster_api_test.go

```diff
@@ -190,6 +207,31 @@ func getSchema(fm kubeFramework, svc *kube_api.Service) (*api_v1.TimeseriesSchem
 	return &timeseriesSchema, nil
 }
 
+func getModelMetrics(fm kubeFramework, svc *kube_api.Service, pod *kube_api.Pod) (*api_v1.MetricResultList, error) {
+	url := fmt.Sprintf("/api/v1/model/namespaces/%s/pod-list/%s/metrics/%s",
```

Comment:
Yeah. That makes sense.

On Tue, Sep 15, 2015 at 1:59 PM, Daniel Martí <notifications@github.com>
wrote:

> In integration/heapster_api_test.go
> <https://github.com/kubernetes/heapster/pull/558#discussion_r39564367>:
>
> > @@ -190,6 +207,31 @@ func getSchema(fm kubeFramework, svc *kube_api.Service) (*api_v1.TimeseriesSchem
> >  	return &timeseriesSchema, nil
> >  }
> >
> > +func getModelMetrics(fm kubeFramework, svc *kube_api.Service, pod *kube_api.Pod) (*api_v1.MetricResultList, error) {
> > +	url := fmt.Sprintf("/api/v1/model/namespaces/%s/pod-list/%s/metrics/%s",
>
> Maybe the issue should be closed once all endpoints are tested. Is that
> what you mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/heapster/pull/558/files#r39564367>.
>